### PR TITLE
re add progress root and popover overlay

### LINKED ISSUE #19236: "cannot read property root of Null" Error when trying to log In via bio verification
### Steps:
1. Onboard enabling bio authentication
2. reopen the app
3. Attempt to log in via bio authentication.

### Actual result:
An error 'cannot read property root of Null' is shown after attempting to log in via bio verification.

https://github.com/status-im/status-mobile/assets/52490791/fb610e67-65f5-4757-bc1f-7c24294c23aa

### Expected result:
No error is shown

### Device:
Pixel 7a, Android 14

### Logs:
[Status-debug-logs (1).zip](https://github.com/status-im/status-mobile/files/14591720/Status-debug-logs.1.zip)

### LINKED ISSUE #19240: "Cannot set property 'reagent-compiler2' of null" Error when blocking a contact
### Steps:
1. UserA adds UserB to contacts
2. UserA or UserB tap 'block' button

### Actual result:
"Cannot set property 'reagent-compiler2' of null" error is shown 

https://github.com/status-im/status-mobile/assets/52490791/39ac5bd4-5d30-437a-8850-c29e13cba2b1

### Expected result:
The blocking drawer should be shown with the ability to block a contact.
image

### ENV:
IOS, Android

### Logs:
[Status-debug-logs (2).zip](https://github.com/status-im/status-mobile/files/14593441/Status-debug-logs.2.zip)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/19236
fixes https://github.com/status-im/status-mobile/issues/19240

regression from https://github.com/status-im/status-mobile/pull/19068

status: ready

### CODE REVIEW

File: src/status_im/navigation/roots.cljs [hunk 1]
```diff
@@ -49,3 +49,19 @@
    {:root {:stack {:children [{:component {:name    :syncing-results
                                            :id      :syncing-results
                                            :options (options/dark-root-options)}}]}}}})
+
+(defn old-roots
```
Comment (flexsurfer) [inline src/status_im/navigation/roots.cljs:53]: could you elaborate on why we still need that?

Comment (Parveshdhull) [reply src/status_im/navigation/roots.cljs:53]: Hi both, biometric login and logout were using an intermediate root screen and removal of that caused #19236 bug.

I can still remove them, means event also which opens this intermediate root. But not sure we want that, as this shows a progress screen.
https://github.com/status-im/status-mobile/blob/develop/src/legacy/status_im/ui/screens/progress/views.cljs

Comment (flexsurfer) [reply src/status_im/navigation/roots.cljs:53]: I mean we should decide if we will continue to use it, it should be in the new roots, etc, overwise remove it

Comment (Parveshdhull) [reply src/status_im/navigation/roots.cljs:53]: https://github.com/status-im/status-mobile/assets/17097240/37cf71e4-ad80-4314-987b-bee13fe84f4a

This is how it looks.

Comment (Parveshdhull) [reply src/status_im/navigation/roots.cljs:53]: > I mean we should decide if we will continue to use it, it should be in the new roots, etc, overwise remove it

Lets keep this screen as temporary until we have something from design team. Should we still move this to new roots, even if its temporary and using legacy screen/comopnents?

Comment (flexsurfer) [reply src/status_im/navigation/roots.cljs:53]: ok let's merge this to fix the bug, and then decide

**Review by flexsurfer — APPROVED** [review]

**Review by clauxx — APPROVED** [review]

**Review by mohsen-ghafouri — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
hi @Parveshdhull thank you for fixes. No issues from my side. Ready to be merged

**Pull request merged**